.github/workflows/ci.yml

```diff
@@ -87,14 +87,21 @@ jobs:
 
   windows-vcpkg:
     name: windows-vcpkg
```

Comment:
oh, can you include the arch in the job name

Reply:
I think it's already visible as `windows-vcpkg (windows-11-arm, arm64)` but sure.